kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+
+  <KPageContainer>
+
+    <h1>{{ $tr('syncData') }}</h1>
+    <span>
+      {{ $tr('access') }}
+      <KButton
+        appearance="basic-link"
+        :text="$tr('learnMore')"
+        @click="displayModal(Modals.PRIVACY)"
+      />
+    </span>
+
+    <CoreTable>
+      <thead slot="thead">
+        <tr>
+          <th>{{ $tr('facility') }}</th>
+        </tr>
+      </thead>
+      <transition-group slot="tbody" tag="tbody" name="list">
+        <tr v-for="facility in facilities" :key="facility.id">
+          <td>
+            <span>
+              <h2 class="name">{{ facility.name }}</h2>
+              <UiIcon v-if="facility.dataset.registered" ref="icon">
+                <mat-svg
+                  name="verified_user"
+                  category="action"
+                  :style="{fill: $themePalette.green.v_500}"
+                />
+              </UiIcon>
+              <KTooltip
+                reference="icon"
+                :refs="$refs"
+              >
+                {{ $tr('registeredAlready') }}
+              </KTooltip>
+            </span>
+            <span class="synced">
+              <span>
+                <template v-if="facility.syncing">
+                  <KCircularLoader class="loader" :size="16" :delay="false" />
+                  {{ $tr('syncing') }}
+                </template>
+                <template v-else>
+                  <template v-if="facility.last_sync_failed">
```

Comment:
right now, we can't verify this information from morango.

Reply:
so, is this always `false`?

Reply:
for now yes, but it was more of a placeholder for when we can actually calculate this boolean.

Reply:
okay. For future reference, recommend either adding a comment indicating that this is stubbed out code and always `false`, or not including it at all.